README.md

```diff
@@ -36,12 +36,20 @@ pip3 install --user .
 ```
 
 ## Using it
-After [installing it](#installing-it), you'll need to edit your Django `settings.py` file
-and particularly the `DATABASES` section to point to an EXISTING database, of the format:
+After [installing it](#installing-it), you'll need to edit your Django `settings.py` file:
+* Add `django-spanner` as the very first app in the `INSTALLED_APPS` section
+* Edit the `DATABASES` section to point to an EXISTING database, of the format:
```

Comment:
Perhaps this should be moved after the INSTALLED_APPS example as it looks odd to have a colon and then what follows isn't DATABASES

Reply:
I'll instead add a new fresh sentence because, in two different they will be confusing.